App.js

```diff
@@ -1,19 +1,17 @@
 import React, { useState, useEffect } from 'react';
 import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
-import LogoSvg from "./LogoSvg";
-import Quotes from "./Quotes";
 import AnimateSvg from './AnimateSvg';
 import AnimateQuotes from './AnimateQuotes';
+import { useIsShake } from './useIsShake';
 
 export default function App() {
+  const [hasInitialTransitionRun, setHasInitialTransitionRun] = useState(false);
```

Comment:
Renamed state variable and setter names for greater code legibility.

Reply:
What does this state describe?